hack/e2e/run.sh

```diff
@@ -25,6 +25,7 @@ source "${BASE_DIR}"/util.sh
 
 DRIVER_NAME=${DRIVER_NAME:-aws-ebs-csi-driver}
 CONTAINER_NAME=${CONTAINER_NAME:-ebs-plugin}
+DRIVER_START_TIME_THRESHOLD=25
```

Comment:
How did we come up with this number? I feel like it should be higher? We can adjust later as we go.

Reply:
I observed on my cluster that usually takes ~15s so I set this number. But makes sense to increase that a little bit in the initial commit, we can adjust this later.

Reply:
Yeah looks like it failed on the CI.

Reply:
I think the delta is the image pull time.. Cold start apparently takes longer than I expected lol